src/services/art_generator.py

```
@@ -0,0 +1,278 @@
+║   ██░▒▓████████▓▒░██         ║
+║  ██░▓█▀╔══╗╔══╗▀█▓░██        ║
+║  █▓▒█╔══║██║══╗█▒▓█         ║
+║  █▓▒█║◆═╚══╝═◆║█▒▓█         ║
+║  ██▓█╚════════╝█▓██         ║
+║   ███▀▀══════▀▀███          ║
+║  ██╱▓▓▓██████▓▓▓╲██         ║
+║ ██▌║▓▓▓▓▀██▀▓▓▓▓║▐██        ║
+║ ██▌║▓▓▓▓░██░▓▓▓▓║▐██        ║
+║  ██╲▓▓▓▓░██░▓▓▓▓╱██         ║
+║   ███▄▄░████░▄▄███          ║
+╚════════════════════════════════╝"""
```

Ruff (0.8.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)